common/src/main/java/hunternif/mc/impl/atlas/ClientProxy.java

```diff
@@ -19,23 +19,27 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.Executor;
+import java.util.function.BiConsumer;
 
 @Environment(EnvType.CLIENT)
 public class ClientProxy implements ResourceReloader {
+    public static BiConsumer<ResourceType, ResourceReloader> SORTED_REGISTER_LISTENER = ReloadListenerRegistry::register;
```

Comment:
Also the name `SORTED_REGISTER_LISTENER` implies way more than there actually is (in case of forge).